classes/bible.lua

```diff
@@ -6,10 +6,6 @@ local plain = require("classes.plain")
 local class = pl.class(plain)
 class._name = "bible"
```

Comment:
This class is a joke.

When will we kill it?

Reply:
> This class is a joke.

Why?

Reply:
Er? What is supposed to be a "bible" class?

Food for thought: I did typeset many bibles (https://github.com/Freely-Given-org/BibleTypesetter/pull/3 -- Still pending any sort of review and approval, which is sad, but anyway) without needing a "bible" **class**.

So what is supposed to be a "bible" class in layman terms?

Reply:
Bible typesetting comes with a standard set of features over and above what most books require. Defaulting to grid typesetting, having headers that keep track of the verse at start and end of pages for use in headers, multiple types of footnotes/cross references, etc. The *bible* class should build on the book class with a few default features available that cover common needs that aren't so common in regular books. Of course you don't have to use it, it's just a starting point.

Additionally it's where I'd put stuff to cover default processing functions for OSIS/USFM/whatever tags that don't have any relevance to non-Bible content.